cmd/cloud/remote/client.go

```diff
@@ -63,9 +63,12 @@ $ ory ...
 		return nil
 	}
 
+	c := retryablehttp.NewClient()
+	c.Logger = nil
```

Comment:
@Demonsthere the CLI printed several debug outputs:

```
2021/05/14 12:07:46 [DEBUG] GET https://api.console.ory.sh/backoffice/token/slug
2021/05/14 12:07:47 [DEBUG] GET https://playground.projects.oryapis.com/api/kratos/admin/identities
```

This really confuses people. We often get issues where people ask about a warning or a log entry because they fear something does not work. They expect to be no output expect the one they ... expect. So when writing these CLI commands we need to ensure that no random debug statements get printed. Also, here, my log level was not set to `DEBUG` which should have been the case if we actually want to see some debugging logs.